### ISSUE TITLE
Restructuring everything!

### ISSUE THREAD

**Issue by richardreeve** [boydorr/EcoSISTEM.jl#14]
I think we need to do a major shift around of everything now we've merged in the SCRC branch:

- [x] Rename Simulation.jl to EcoSISTEM.jl
- [x] Rename main to dev
- [x] Only allow PR merges to dev if approved by one of the dev team
- [x] Rename biodiversity to main
- [x] Create @boydorr/ecosistem-admins group
- [x] Only allow PRs to main when we are going to make a release (approved by > 1 of the admin team?)
- [x] Add in `.zenodo.json` file to capture metadata (can be mostly copied from Phylo)
- [x] Add an automatic trigger to Zenodo to capture tagged releases of EcoSISTEM
- [x] Resolve warnings in tests
- [x] Link to contribution agreement from README and/or docs
- [x] Release 0.1 of the package based on main (ex biodiversity)
- [x] Fix license by removing first line and quotes, so GitHub recognises the license correctly
- [x] Update to 0.1.1 with minor fixes identified post 0.1 release
- [x] Permanent fix to Diversity incompatibility
- [x] Update to 0.1.2
- [x] Merge `main` from 0.1.2 into `dev`
- [x] Add SCRC authors to the `dev` `.zenodo.json` file

**Comment by gottacatchenall** [comment]
tagging @tpoisot here as he mentioned he has done renaming before

**Comment by richardreeve** [comment]
@claireh93 I've added in a few more bullets into the list above I'm afraid :)

**Comment by richardreeve** [comment]
@claireh93 Just a quick note that I've added a few more things to this list... all trivial though.

**Comment by richardreeve** [comment]
Merging main into dev is being done in a PR in ScottishCovidResponse/EcoSISTEM.jl#95. After that we then need to add the SCRC authors to the .zenodo.json file.

**Comment by richardreeve** [comment]
@claireh93 I'd be happy if we moved the last couple of items here to a new issue (on the new project board?) and close this if it makes sense to you? I think the restructure is now done, and we're just waiting on external stuff now...

**Comment by claireh93** [comment]
Yes sounds good to me!
